### ISSUE TITLE
SEGM Detector (SEGS) causes AttributeError: 'NoneType' object has no attribute 'data'

### ISSUE THREAD

**Issue by ghost** [ltdrdata/ComfyUI-Impact-Pack#204]
For some reasons this broke a few days ago:
![Screenshot 2023-10-07 154952](https://github.com/ltdrdata/ComfyUI-Impact-Pack/assets/146747696/052f6c62-3aae-4c02-bdea-bcc4c058c44b)

And it's not like one can make many mistakes here, give it an image, give it a SEGM detector and done.
Image is directly fed from a LoadImage node:
![Screenshot 2023-10-07 155451](https://github.com/ltdrdata/ComfyUI-Impact-Pack/assets/146747696/0da76642-c631-4f2e-a2ed-1d19a13013ac)

All Custom_Nodes are updated through the ComfyManager, as is ComfyUI itself. Even checked ImpactPack with git pull, "Already up-to-date"

**Comment by ltdrdata** [comment]
Check startup message.
```
### Loading: ComfyUI-Impact-Pack (V4.16.3)
### Loading: ComfyUI-Impact-Pack (Subpack: V0.2.3)
```

Recent, Impact Pack version is 4.16.3
Subpack version should V0.2.3.

My guess is that the files in your `impact_subpack` directory under your `impact_subpack` path have been modified, which might be why the update failed. Please try running `git reset --hard && git pull` directly in the `custom_nodes/ComfyUI-Impact-Pack/impact_subpack` directory."

**Comment by ghost** [comment]
> Check startup message.
> 
> ```
> ### Loading: ComfyUI-Impact-Pack (V4.16.3)
> ### Loading: ComfyUI-Impact-Pack (Subpack: V0.2.3)
> ```
> 
> Recent, Impact Pack version is 4.16.3 Subpack version should V0.2.3.
> 
> My guess is that the files in your `impact_subpack` directory under your `impact_subpack` path have been modified, which might be why the update failed. Please try running `git reset --hard && git pull` directly in the `custom_nodes/ComfyUI-Impact-Pack/impact_subpack` directory."

Sadly that didn't work:
![Screenshot 2023-10-08 121213](https://github.com/ltdrdata/ComfyUI-Impact-Pack/assets/146747696/8e8966da-2f14-4422-a795-0a598cd655f7)

Versions are correct:
![Screenshot 2023-10-08 121130](https://github.com/ltdrdata/ComfyUI-Impact-Pack/assets/146747696/31e3c1f4-dfa6-49d1-9f21-f1ae60cb33a9)

I even did a complete delete and did the https://github.com/ltdrdata/ComfyUI-Impact-Pack#installation steps, still the same error. 

Really don't understand why, as it worked fine a few days ago, so some update did come through to break it? Did something in ComfyUI change for it to break or what happened?

**Comment by ltdrdata** [comment]
And then what is your ComfyUI version.

**Comment by ghost** [comment]
> And then what is your ComfyUI version.

Updated through manager and checked again with git pull:
![image](https://github.com/ltdrdata/ComfyUI-Impact-Pack/assets/146747696/69169bf1-9142-4743-ac1b-91bcd3b71625)

Don't know if there is an obvious version number somewhere.

Edit: found it:
### ComfyUI Revision: 1544 [1f2f4eaa] | Released on '2023-10-08'

**Comment by ltdrdata** [comment]
Send me the your workflow. I'll check it.

**Comment by ghost** [comment]
[SEGMtestWorkflow.txt](https://github.com/ltdrdata/ComfyUI-Impact-Pack/files/12840524/SEGMtestWorkflow.txt)

here is a super reduced workflow with the minimum necessary.

and the full one just incase:

[FullWorkflowSEGM.txt](https://github.com/ltdrdata/ComfyUI-Impact-Pack/files/12840546/FullWorkflowSEGM.txt)

**Comment by ltdrdata** [comment]
gotcha... This is occurs when there is no detected face.

**Comment by ltdrdata** [comment]
That is fixed on 702cf6a39c1d49651837442192a041dd7f0fb505
